src/python_testing/TC_IDM_1_4.py

```diff
@@ -79,7 +79,7 @@ async def test_TC_IDM_1_4(self):
         asserts.assert_greater_equal(len(list_of_commands_to_send), 2,
```

Comment:
Adding blocking comment to make sure https://github.com/project-chip/connectedhomeip/pull/31455 is merged first